.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+### Which issue this PR addresses:
+
+<!--
+Please include a link to the VSTS work item as well as any GitHub issues.
+
+Usage: `Fixes #<GitHub issue number>`, or `Fixes (paste link of issue)`.
+-->
+Fixes
+
+### What this PR does / why we need it:
+
+<!--
+Include a brief summary of what the PR is intended to accomplish and how the PR
+does it. (2-3 sentences)
+-->
+
+### Test plan for issue:
+
+<!--
+How did you test that this PR works?
+
+- Are there unit tests?
+- Are there integration/e2e tests?
+- If it is not possible to write automated tests, explain why and document how
+  to manually test and verify the feature.
+-->
+
+### Is there any documentation that needs to be updated for this PR?
+
```

Comment:
I suspect this is mostly for big features, and we will mostly be deleting the section

Reply:
I don't think that's necessarily the case. For example, we should be maintaining documentation on available admin actions; if a new one is added, the docs should be updated. If we add a new alert in here, same thing.